script.test.js

```diff
@@ -1,13 +1,18 @@
-const weather = require('./script');
+/**
+ * @jest-environment jsdom
+ */
```

Comment:
`@jest-environment jsdom` has to be added to the file to replace `node` environment which is default. Check the [jest doc](https://jestjs.io/docs/configuration#testenvironment-string)